src/Templates/src/Microsoft.Maui.Templates.csproj

```diff
@@ -31,7 +35,7 @@
 
   <ItemGroup>
     <_TemplateFile Update="@(_TemplateFile)">
-      <NewFileName>$(IntermediateOutputPath)\$([System.IO.Path]::GetFileName('%(Identity)').Replace('.in',''))</NewFileName>
+      <NewFileName>$(IntermediateOutputPath)\$([System.IO.Path]::GetDirectoryName('%(Identity)'))\$([System.IO.Path]::GetFileName('%(Identity)').Replace('.in',''))</NewFileName>
```

Comment:
I needed to unique-ify the names because I use the exact same file names for maui-blazor as are used in maui-mobile.